packages/react-composites/tests/browser/call/hermetic/RoomsCallScreen.test.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { buildUrlWithMockAdapter, defaultMockCallAdapterState, defaultMockRemoteParticipant, test } from './fixture';
+import { expect } from '@playwright/test';
+import { dataUiId, stableScreenshot, waitForSelector } from '../../common/utils';
+import { IDS } from '../../common/constants';
+
+test.describe('Rooms CallScreen tests for different roles', async () => {
+  test('All CallControls are enabled for Presenter', async ({ page, serverUrl }) => {
+    const initialState = defaultMockCallAdapterState();
+    await page.goto(buildUrlWithMockAdapter(serverUrl, initialState, { role: 'Presenter' }));
+    await waitForSelector(page, dataUiId(IDS.videoGallery));
+    expect(await stableScreenshot(page, { dismissTooltips: true })).toMatchSnapshot('rooms-call-screen-presenter.png');
+  });
+
+  test('All CallControls are enabled for Attendee', async ({ page, serverUrl }) => {
+    const initialState = defaultMockCallAdapterState();
+    await page.goto(buildUrlWithMockAdapter(serverUrl, initialState, { role: 'Attendee' }));
+    await waitForSelector(page, dataUiId(IDS.videoGallery));
+    expect(await stableScreenshot(page, { dismissTooltips: true })).toMatchSnapshot('rooms-call-screen-attendee.png');
+  });
+
+  test('Only few CallControls are enabled for Consumer', async ({ page, serverUrl }) => {
+    const initialState = defaultMockCallAdapterState();
+    await page.goto(buildUrlWithMockAdapter(serverUrl, initialState, { role: 'Consumer' }));
+    await waitForSelector(page, dataUiId(IDS.videoGallery));
+    expect(await stableScreenshot(page, { dismissTooltips: true })).toMatchSnapshot('rooms-call-screen-consumer.png');
```

Comment:
I know this is just a hermetic fake-data artifact, but is odd that the participant is stuck in the unmuted state:
![image](https://user-images.githubusercontent.com/2684369/185197674-9fdbe187-e7cc-4afc-a869-24fba35a6546.png)

Reply:
(I don't think we can do anything about this here, the call state should just always be muted when in this role)

Reply:
Yeah, we are using same default state in all mock tests and user is unmuted by default in mock state, that is not the case in actual call though :)